docs/guides/getting-started/quick-start/vite.md

```diff
@@ -0,0 +1,90 @@
+import TauriInit from './\_tauri-init.mdx';
+import Commands from './\_commands.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Vite
+
+This guide will walk you through creating your first Tauri app using the frontend build tool [Vite].
+
+<TauriInit
+  webAssets="To be compatible with Vite's default config we will answer this with `../dist`."
+  devServer="Because Vite's development server will listen on port 3000 by default we will answer this question with `http://localhost:3000`."
+/>
```

Comment:
For vite, it is better to create the vite project first then add tauri to it, because create-vite won't bootstrap a project when the project dir is not empty.

Reply:
I think this would probably be the way across most frameworks, especially if you're using npm

Reply:
agreed, I think only vanilla could start with `tauri init` then front end.

Reply:
@JonasKruckenberg is that something you can look at when you're doing the screenshots and code example for Vite?

Reply:
Jup!